src/handlers/package_handler.js

```diff
@@ -643,7 +643,7 @@ async function postPackagesVersion(req, res) {
   // Else we will continue, and trust the name provided from the package as being accurate.
   // And now we can ensure the user actually owns this repo, with our updated name.
 
-  const gitowner = await git.ownership(user.content, packJSON.name);
+  const gitowner = await git.ownership(user.content, packExists.metadata.repository.replace("https://github.com/", ""));
```

Comment:
the `metadata.repository` field here is the one contained in the `package.json`? Because in my case it is in the format `git@github.com:<org>/<repo>.git`, so this replace won't have effect

Reply:
Good call, thanks for pointing this out. I'll go ahead and get those other updates again and go over this section with a fine toothed comb looking for any assumptions like this elsewhere

Reply:
> is the `metadata.repository` field here the one contained in the `package.json`? Because in my case it is in the format `git@github.com:<org>/<repo>.git`, so this replace won't have effect

So the following used as a reference is an old format?

https://github.com/pulsar-edit/package-backend/blob/main/docs/resources/package_object_full.json#L4424